tests/System/client_multivo.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+testdir=`echo $PWD`
```

Comment:
I might be missing something but:
```suggestion
testdir=$PWD
```